client/dashboard/sites/settings-static-file-404/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { DataForm } from '@automattic/dataviews';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import {
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	__experimentalVStack as VStack,
+	Card,
+	CardBody,
+	Button,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { settings } from '@wordpress/icons';
+import { store as noticesStore } from '@wordpress/notices';
+import { useState } from 'react';
+import { siteQuery, siteStaticFile404Query, siteStaticFile404Mutation } from '../../app/queries';
+import { Callout } from '../../components/callout';
+import PageLayout from '../../components/page-layout';
+import SettingsPageHeader from '../settings-page-header';
+import calloutIllustrationUrl from './callout-illustration.svg';
+import type { Site } from '../../data/types';
+import type { Field } from '@automattic/dataviews';
+
+export function canSetStaticFile404Handling( site: Site ) {
+	return site.is_wpcom_atomic;
+}
+
+const fields: Field< { setting: string } >[] = [
+	{
+		id: 'setting',
+		label: __( 'Response' ),
```

Comment:
Should this be "Server Response"? It isn't clear until I read the items what 'Response' is referring to.